presentations/en/SLF.md

```diff
@@ -7,39 +11,40 @@ ESDC IT Strategy
 
 ---
 
-## Agenda
-
-- The need for Strategy
-- Approach
-- Benefits
-
----
-
 <img src="assets/images/slf-presentation-smartphone-adoption-infographic-v2-1007.png" alt="Graph of smartphone adoption rate: 1 billion users took 16 years but only 4 years to reach 2 billion" height="85%" width="85%">
 
 Note:
 
 - Change is already here, we are experiencing it right now
-- The upcoming Policy on Service and Digital, OneGC, and the GC Digital Stanards, is the GC catching up with citizens expectations
-- Banks are training investment bankers to code. Source: https://www.techrepublic.com/article/why-big-banks-are-requiring-workers-to-learn-coding/
-- Cars now have hundred millions lines of codes. Source: https://www.technologyreview.com/s/508231/many-cars-have-a-hundred-million-lines-of-code/
 
 --
 
-insert pic of amount of github contribution over the years showing pace of software change
+![Number of CVEs over time](./assets/images/slf-presentation-cve.png)
+
+_source: cvedetails.com_
 
 Note:
 
-- Using Github as a source to measure the pace of software change, we can see that it's exponential
-- This is a glimps of how addressing software patching, versioning, and overall intergration complexity needs to be addressed
+- This graph represents the number of Common Vulnerabilities and Exposures (CVE) over a 20-year timespan
+- Security vulnerabilities and exposures are also increasing exponentially
+- Source: https://www.cvedetails.com/
 
 --
 
 ![Change is the only constant, and its rate is accelerating](./assets/images/arrow_animated.gif)
 
 Note:
 
-- Change is the only constant, and its rate is accelerating
+- The upcoming Policy on Service and Digital, OneGC, and the GC Digital Stanards, is the GC catching up with citizens expectations
+- Banks are training investment bankers to code. From Financial Times and Wall Street Journal.
+  - E.g. Citigroup traders get a 3-day Python course. JPMorgan Chase 300 analysts mandatory Python programming courses.
+  - Source: https://www.techrepublic.com/article/why-big-banks-are-requiring-workers-to-learn-coding/
+- Cars now have hundred million lines of codes.
+  - Source: https://www.technologyreview.com/s/508231/many-cars-have-a-hundred-million-lines-of-code/
```

Comment:
Perhaps we could use a book reference over an online article?

"Windows operating system has 60 million lines of code, and in 2010 cars already contained around one hundred million." 
Project to Product, Mik Kersten

Reply:
will push the update. Great notes to speak about